04-using-github-with-the-platform.Rmd

```diff
@@ -220,35 +224,41 @@ A Personal Access Token (PAT) is provided by GitHub through each user's profile,
   2. Select **_Generate new token_** and select the **scope** of access you want to grant using this token. **IMPORTANT: Unless you have specific reasons to need more control, just select the first group, `repo`--granting other rights can be dangerous if your PAT falls into the wrong hands, allowing someone else to irrevocably delete your repos, read and change your user profile settings, or even access billing information.**
   ![](images/pat/pat-scope.png)
   
-  3. Copy your PAT into a `.txt` file that you can keep somewhere securely. You won't be able to come back to this page to see this PAT again, and will have to generate a new one if you lose track of it.
+  3. Copy your PAT to the clipboard, for immediate use - see the next section.
+  
+     Note: GitHub won't show this PAT to you ever again BUT you shouldn't save it in a file or anything. If you were to have passwords in plain text lying around your hard disk then it becomes a security hazard. If you need a PAT in future (eg you wiped your .simply create a new one, and invalidate the previous one.
   ![](images/pat/pat-copy.png)
   
-  ### Using a PAT to Authenticate in R/RStudio
+### Using a PAT to Authenticate in R/RStudio
 
 A safe, fast, and reliable method is to store the PAT in the so-called **system environment**.
 This is an area where basic, 'long-term' variables are kept, like your Operating System details, that do not change, even as you switch between analyses, sessions or projects.
-Conveniently, most [`r-lib`-style](https://github.com/r-lib) packages (those built by the inimitable Hadley Wickham and friends--for most people a backbone of R infrastructure) including `remotes` and `devtools`, will look for a PAT in the system environment by default, under the name `GITHUB_PAT`, so no modifications need be made to any project code once a user has registered a PAT in this location.
+Conveniently, most [`r-lib`-style](https://github.com/r-lib) packages (those built by Hadley Wickham and friends--for most people a backbone of R infrastructure) including `remotes` and `devtools`, will look for a PAT in the system environment by default, under the name `GITHUB_PAT`, so no modifications need be made to any project code once a user has registered a PAT in this location.
```

Comment:
I don't know that we even need to namecheck Hadley Wickham, do we? Can we just say something like "those that are part of the tidyverse, for many people a backbone of R infrastructure"?

(☝️ assuming what I've written about the Tidyverse is actually true)

Reply:
My only intention here was that I'm not sure people know what the tidyverse is but often people are directed to (or at least I direct people to) Hadley's 'R for Data Science' so they might better relate it to their work. I promise it's not a personal obsession with HW, promise...

Reply:
I've trimmed out the tidyverse stuff as it is a bit off track. It now says `[`GITHUB_PAT` env variable] is where common R packages (e.g. remotes and devtools) will look for it.`